src/Tool/DataObjectLoader.php

```diff
@@ -60,7 +60,7 @@ private function getAttributeNameFromParts(array $objectBrickParts,
 
     public function loadByAttribute(string $className,
                                     string $attributeName,
-                                    string $identifier,
+                                    ?string $identifier,
```

Comment:
```suggestion
                                    string $identifier = '',
```

Reply:
I created a new PR, but why this change? default value identifier seems odd / unneeded.

Reply:
@cancan101 Thanks for creating the new PR. The change of the type of a parameter from a public function is a BC-Break, because the class isn't internal or final

Reply:
@cancan101 When we change `string $identifier` to `?string $identifier` and someone extends from `DataObjectLoader` and override `loadByAttribute` they also need to add the `?` to  `string $identifier`. So then it's a BC-Break.

Reply:
Please ignore this PR. I understand it's a breaking change which is why I open the other PR.